zutils.cc

```diff
@@ -23,7 +23,65 @@ void ZBackup::backupFromStdin( string const & outputFileName )
 {
   if ( isatty( fileno( stdin ) ) )
     throw exWontReadFromTerminal();
+  backupFromFileHandle( "stdin", stdin, outputFileName );
+}
+
+/// Backs up the data from a file
+void ZBackup::backupFromFile( string const & inputFileName, string const & outputFileName )
+{
+  FILE* f = fopen( inputFileName.c_str(), "r" );
+  if ( NULL == f )
```

Comment:
You are not handling special files in any way.
Adding symlink to source directory causes exInputError - that wasn't obvious for end-users.